extensions/gdx-freetype/src/com/badlogic/gdx/graphics/g2d/freetype/FreeTypeFontGenerator.java

```diff
@@ -345,6 +345,8 @@ public FreeTypeBitmapFontData generateData (FreeTypeFontParameter parameter, Fre
 			}
 			ownsAtlas = true;
 			packer = new PixmapPacker(size, size, Format.RGBA8888, 1, false, packStrategy);
+			packer.setCurrentTransparentColor(parameter.color);

```

Comment:
If a border and/or shadow is being used, it might make sense to use one of those instead of the glyph color.